test/Swashbuckle.AspNetCore.SwaggerGen.Test/SwaggerGenerator/SwaggerGeneratorTests.cs

```diff
@@ -1723,15 +1723,69 @@ public void GetSwagger_Works_As_Expected_When_FromForm_Attribute_With_SwaggerIgn
+                                    new DefaultMetadataDetails(ModelMetadataIdentity.ForProperty(propertyEnum, typeof(IntEnum), typeof(TypeWithDefaultAttributeOnEnum)), ModelAttributes.GetAttributesForProperty(typeof(TypeWithDefaultAttributeOnEnum), propertyEnum)));
+
+            var propertyEnumArray = typeof(TypeWithDefaultAttributeOnEnum).GetProperty(nameof(TypeWithDefaultAttributeOnEnum.EnumArrayWithDefault));
+            var modelMetadataForEnumArray = new DefaultModelMetadata(
+                                    new DefaultModelMetadataProvider(new FakeICompositeMetadataDetailsProvider()),
+                                    new FakeICompositeMetadataDetailsProvider(),
+                                    new DefaultMetadataDetails(ModelMetadataIdentity.ForProperty(propertyEnumArray, typeof(IntEnum[]), typeof(TypeWithDefaultAttributeOnEnum)), ModelAttributes.GetAttributesForProperty(typeof(TypeWithDefaultAttributeOnEnum), propertyEnumArray)));
+            var subject = Subject(
+               apiDescriptions:
+               [
+                   ApiDescriptionFactory.Create<FakeController>(
+                        c => nameof(c.ActionHavingFromFormAttributeWithSwaggerIgnore),
+                        groupName: "v1",
+                        httpMethod: "POST",
+                        relativePath: "resource",
+                        parameterDescriptions: new[]
+                        {
+                            new ApiParameterDescription
+                            {
+                                Name = nameof(TypeWithDefaultAttributeOnEnum.EnumWithDefault),
+                                Source = BindingSource.Query,
+                                Type = typeof(IntEnum),
+                                ModelMetadata = modelMetadataForEnum
+                            },
+                            new ApiParameterDescription
+                            {
+                                Name = nameof(TypeWithDefaultAttributeOnEnum.EnumArrayWithDefault),
+                                Source = BindingSource.Query,
+                                Type = typeof(IntEnum[]),
+                                ModelMetadata = modelMetadataForEnumArray
+                            }
+                        })
+               ],
+               schemaFilters: [new TestEnumSchemaFilter()]
+           );
+            var document = subject.GetSwagger("v1");
+
+            var operation = document.Paths["/resource"].Operations[OperationType.Post];
+            Assert.NotEmpty(operation.Parameters);
+            Assert.Equal(nameof(TypeWithDefaultAttributeOnEnum.EnumWithDefault), operation.Parameters[0].Name);
+            Assert.NotEmpty(operation.Parameters[0].Description);
+            Assert.Equal(nameof(TypeWithDefaultAttributeOnEnum.EnumArrayWithDefault), operation.Parameters[1].Name);
+            Assert.NotEmpty(operation.Parameters[1].Description);
+
```

Comment:
```suggestion
```

Reply:
I am not very familiar with IntegrationTest. If I add a controller in the Basic webSite and the EnumSchemaFilter then all is OK right?

Reply:
If you take a look at the recently added verification tests, some thing like that.

Reply:
I think right now is done :). I change the test to the Verify part :)